model/project_ref_test.go

```diff
@@ -61,13 +61,13 @@ func TestFindMergedProjectRef(t *testing.T) {
 	assert.NoError(t, v1.Insert())
 
 	parserProject := &ParserProject{
-		Id:                 "ident",
-		DeactivatePrevious: utility.TruePtr(),
-		TaskAnnotationSettings: &evergreen.AnnotationsSettings{
-			FileTicketWebhook: evergreen.WebHook{
-				Endpoint: "random2",
-			},
-		},
+		Id: "ident",
+		//DeactivatePrevious: utility.TruePtr(),
```

Comment:
Why is this commented out?

Reply:
Ah,  a byproduct of previous debugging. Uncommented